lib/ansible/plugins/inventory/vultr.py

```diff
@@ -121,9 +153,17 @@ def _retrieve_servers(api_key):
         raise AnsibleError("Error while fetching %s: %s" % (api_url, to_native(e)))
 
 
-class InventoryModule(BaseInventoryPlugin):
+class InventoryModule(BaseInventoryPlugin, Constructable):
+
     NAME = 'vultr'
 
+    def verify_file(self, path):
```

Comment:
Any reason to enforce that ? I know it's been used in other inventory plugins, but shouldn't we let user use the file name they want without enforcing it ? Plus most plugins do not specify in their documentation what the name should look like, leading to few mins of troubleshooting before "grasping" it should follow a specifal pattern

Reply:
considering the the problems 'unspecified' brings to the table, i would insist, yes

all other plugins should have this clearly documented, if not, its a bug in that plugin.